internal/wazeroir/compiler.go

```diff
@@ -847,14 +845,12 @@ operatorSwitch:
 		if isVector {
 			c.emit(
 				&OperationPick{Depth: 1, IsTargetVector: isVector},
-				&OperationSwap{Depth: depth + 2, IsTargetVector: isVector},
-				&OperationDrop{Depth: &InclusiveRange{Start: 0, End: 1}},
```

Comment:
`OperationSwap` was always followed by Drop, and because of that, we had a redundant register allocation in Swap. That is what this commit resolves.